src/snowflake/snowpark/functions.py

```diff
@@ -867,7 +869,7 @@ def create_map(
     col = object_construct_keep_null(*cols)
 
     # Alias to create_map
```

Comment:
nit: should we add a helper for that, that uses internally _set_fn_name?

Reply:
Done.